src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -175,3 +175,6 @@
           (count (get-in entity ["attributes" (set-type->membership-attribute entity-type)]))
           ;; something nonsensical has been selected, submission will probably fail anyway:
           :else 1)))
+
+(defn url-to-doid [url]
+   (str "DOID: " (second (clojure.string/split url #"DOID_"))))
```

Comment:
add a TODO comment: when we support multiple ontologies, we'll need to look for some string other than "DOID".